server/index.js

```diff
@@ -2,7 +2,6 @@ const express = require('express');
 const bodyParser = require('body-parser');
```

Comment:
express comes with bodyParser library included as of  v4.16.

Uninstall bodyParser and use the included middleware.

```
app.use(express.json());
```